modules/seqexec/server/src/main/scala/seqexec/server/gmos/GmosSouth.scala

```diff
@@ -44,6 +42,7 @@ final case class GmosSouth[F[_]: Concurrent: Timer: Logger] private (
       config.extractInstAs[SouthTypes#FPU](FPU_PROP_NAME)
     def extractStageMode(config: CleanConfig): Either[ConfigUtilOps.ExtractFailure, GmosSouthType.StageModeSouth] =
       config.extractInstAs[SouthTypes#GmosStageMode](STAGE_MODE_PROP)
+    def isCustomFPU(config: CleanConfig): Boolean = extractFPU(config).map(_.isCustom()).getOrElse(false)
```

Comment:
Will require an ODB update to deploy in production?  (I know you updated the code in a separate PR, just wondering if we need to deploy an updated ODB to use it.)

Reply:
Not really, I exported the new code as a lib but since we are converting xml to ocs code in the seqexec it should work

My local tests worked fine with an older odb

Reply:
Oh, that's right.  :+1: